src/chopper_layout.cpp

```diff
@@ -16,79 +16,132 @@ void set_up_subparser_layout(seqan3::argument_parser & parser, chopper::layout::
+                    "preferred. Attention: Only possible if the directory [INPUT-PREFIX]_sketches is present.");
+
+    parser.add_flag(config.rearrange_bins,
+                    '\0', "rearrange-user-bins",
+                    "As a preprocessing step, rearranging the order of the given user bins based on their sequence "
+                    "similarity may lead to favourable small unions and thus a smaller index."
+                    "Attenation: Also enables --estimate-union and is only possible if the directory "
+                    "[INPUT-PREFIX]_sketches is present.");
+
+    parser.add_subsection("Parameter Tweaking:");
+    // -----------------------------------------------------------------------------------------------------------------
+    parser.add_option(config.alpha,
+                      '\0', "alpha",
+                      "The layout algorithm optimizes the space consumption of the resulting HIBF but currently has no "
+                      "means of optimizing the runtime for querying such an HIBF. In general, the ratio of merged bins "
+                      "and split bins influences the query time because a merged bin always triggers another search on "
+                      "a lower level. To influence this ratio, alpha can be used. The higher alpha, the less merged "
+                      "bins are chosen in the layout. This improves query times but leads to a bigger index.",
+                      seqan3::option_spec::advanced);
+
+    parser.add_option(config.max_ratio,
+                      '\0', "max-rearrangement-ratio",
+                      "When the option --rearrange-user-bins is set, this option can influence the rearrangement "
+                      "algorithm. The algorithm only rearranges the order of user bins in fixed intervals. The higher "
+                      "--max-rearrangement-ratio, the larger the intervals. This potentially improves the layout, but "
+                      "increases the runtime of the layout algorithm.",
+                      seqan3::option_spec::standard,
+                      seqan3::arithmetic_range_validator{0.0, 1.0});
 
-    parser.add_flag(config.output_statistics, '\0', "output-statistics",
-                    "Print additional statistics and information to the command line (std::cout).",
+    parser.add_subsection("Special options");
+    // -----------------------------------------------------------------------------------------------------------------
+    parser.add_flag(config.determine_num_bins,
+                    '\0', "determine-best-tmax",
+                    "When this flag is set, the program will compute multiple layouts for tmax in "
+                    "[64 , 128, 256, ... , tmax] as well as tmax=sqrt(number of user bins)."
+                    "The estimated query cost and index size is used to determine the best tmax for your data. "
```

Comment:
@eseiler  Isn't this already what we talked about?

Reply:
Actually yes, but it could highlight the difference to the DP, something like:
`The layout algorithm itself only optimizes the space consumption. When determining the best layout, we keep track of the average number of queries needed to traverse the layout...`